src/classes/Bot.ts

```diff
@@ -34,11 +34,10 @@ class Bot {
 		password: process.env.MINECRAFT_PASSWORD,
 		host: "mc.hypixel.net",
 		auth: "microsoft",
-		version: "1.16.4",
+		version: "1.8.9",
 		logErrors: true,
 		hideErrors: true,
-		checkTimeoutInterval: 30000,
-		defaultChatPatterns: false,
```

Comment:
Why have you removed this config option? This option is present to prevent listeners from being created unnecessarily - we do not use the defaults. See [here](https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#mineflayercreatebotoptions).